klayout_package/python/kqcircuits/chips/__init__.py

```diff
@@ -22,5 +22,5 @@
 there are methods to place launchers and some test structures. After adding other elements to the chip, a ground grid
 which avoids the elements can be produced.
 
-See ``kqcircuits.elements`` documentation for a general description of elements.
+See `kqcircuits.elements <https://iqm-finland.github.io/KQCircuits/api/kqcircuits.elements.html>`_ documentation for a general description of elements.
```

Comment:
This could be done without hard-coding the URL: 

```suggestion
See :mod:`.kqcircuits.elements` documentation for a general description of elements.
```